client/js/upload.js

```diff
@@ -141,7 +141,8 @@ class Uploader {
 		if (
 			store.state.settings.uploadCanvas &&
 			file.type.startsWith("image/") &&
-			!file.type.includes("svg")
+			!file.type.includes("svg") &&
+			!file.type.includes("gif")
```

Comment:
Can't this just be `file.type === 'image/gif'`?

Reply:
I just went with the way it has been done for SVGs 🤷 , either way would work. Should I change it?

Reply:
Sure. The svg uses `includes` because the mime type is incosistent as far as I remember.

Reply:
Not according to https://www.iana.org/assignments/media-types/media-types.xhtml, shall I change both?

Reply:
I remember there was something specific to svg that had to be this way, I can't remember now though, so just keep it.